x/evm/types/msg_evm.go

```diff
@@ -432,11 +439,19 @@ func (msg *MsgEthereumTx) RawSignatureValues() (v, r, s *big.Int) {
 // From loads the ethereum sender address from the sigcache and returns an
 // sdk.AccAddress from its bytes
 func (msg *MsgEthereumTx) AccountAddress() sdk.AccAddress {
-	return ethcmn.FromHex(msg.GetFrom())
+	if msg.addr == emptyEthAddr {
+		return ethcmn.FromHex(msg.GetFrom())
+	} else {
+		return msg.addr[:]
```

Comment:
```suggestion
		return msg.addr.Bytes()
```
If addr.Bytes() not increase the overhead so much, I suggest use this function, otherwise keep [:]

Reply:
```go
func (a Address) Bytes() []byte { return a[:] }
```
`Bytes` is a value receiver function, and if return Bytes(), this value will escape to heap, each function call will have one more object allocation